packages/model-viewer/src/three-components/ARRenderer.ts

```diff
@@ -569,19 +571,18 @@ export class ARRenderer extends EventDispatcher {
     } else if (fingers.length === 2) {
       box.show = true;
       this.isTwoFingering = true;
-      const {separation} = this.fingerPolar(fingers);
-      this.firstRatio = separation / scene.scale.x;
```

Comment:
> 
> 
> I'm not sure if this is some intended behavior which is being erratic but it seems like a bug to me.
> When testing the [demo linked above](https://demos.r2u.io/model-viewer/index.html) by @FelipeR2U, sometimes I am able to double tap box corners which are diagonally opposite and make the model do a 180. I am able to reproduce on `master` but it's far less common.

Well noticed, I had the same issue sometimes and realized I should probably rollback this change so that `this.lastAngle` is updated.